src/test/java/com/conveyal/datatools/manager/controllers/api/UserControllerTest.java

```diff
@@ -154,6 +340,16 @@ public void canCreateUser() throws IOException {
      */
     @Test
     public void canReturnMeaningfulAuth0Error() throws IOException {
+
+        String url = USERS_API_PATH + "?sort=email%3A1&per_page=10&page=0&include_totals=false&search_engine=v3&q=email%3A" + URLEncoder.encode(emailForExistingAccount, "UTF-8") + "*";
```

Comment:
This common set of query params should be placed in a constant and reused.

Reply:
Updated.